forte/processors/elastic_search_query_creator.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=attribute-defined-outside-init
+from typing import Dict, Any
+
+from texar.torch.hyperparams import HParams
+
+from forte.common.resources import Resources
+from forte.data import MultiPack
+from forte.processors.base import QueryProcessor
+
+__all__ = [
+    "ElasticSearchQueryCreator"
+]
+
+
+class ElasticSearchQueryCreator(QueryProcessor[MultiPack]):
+    r"""This processor creates a Elasticsearch query and adds it as Query entry
+    in the data pack. This query will later used by a Search processor to
+    retrieve documents."""
+
+    # pylint: disable=useless-super-delegation
+    def __init__(self) -> None:
+        super().__init__()
+
+    def initialize(self, resources: Resources, configs: HParams):
+        self.resource = resources
+        self.config = configs
+
+    def _build_query(self, text: str) -> Dict[str, Any]:
```

Comment:
Add some docstring here to teach users to extend this method in order to create more complex queries.